### ISSUE TITLE
not have double artist for an artist with a feat

### ISSUE THREAD

**Issue by legacychimera247** [equeim/unplayer#30]
as of now if you have an album with a song featuring someone that song will only be shown under it's appropriate artist

eg a song by U2 feat bono in the album u2bono will only be present in that album from the artist u2 feat bono and not in the same album by u2

it would be nice to eliminate that feat artist in the list and just have the song in the appropriate album with just a "feat bono" on the artist, like stock player is doing.

you may have a look, if you are a linux user, at how the lollypop player works, for better understanding

**Comment by equeim** [comment]
IIRC, I took this case into account when writing app. It can be either a bug in code logic or incorrect tags of music file (or limitations of file's format). Could you send me an example of file to reproduce the bug? Unfortunately, I won't be able to continue working on the project until end of April.

**Comment by legacychimera247** [comment]
i know you wont be able to ced till spring but anyway i thought worth reporting...

anyway, format is plain mp3 or any other format...i'll give you examples...
this is boxes from goo goo dolls in stock music player
https://1drv.ms/i/s!AlyMkCypv00qgsUjA14cImiPoLNL-w

this is the the album in unplayer
https://1drv.ms/i/s!AlyMkCypv00qgsUk1I7IyxMSaks3wg

as you may see the song Flood which actually as a feat. is not present, it's only listed on it's own artist
https://1drv.ms/i/s!AlyMkCypv00qgsUlbwbhY7nD2iCqtA

it would be nice to have all under the same album like jolla stock player

and by the way, happy new year :)

**Comment by ghost** [comment]
You can solve this yourself by setting the album artist tag to 'Goo Goo Dolls', and fixing the artist tag of 'Goo Goo Dolls feat. Sydney Sierota' to a double artist tag: 'Goo Goo Dolls' and 'Sydney Sierota'. You can easily do this with Ex Falso (https://github.com/quodlibet/quodlibet), not sure how to do it with other tag editors. You will be able to find all 11 tracks under Goo Goo Dolls, and 'Flood' under Sydney Sierota, too.

**Comment by legacychimera247** [comment]
Nope...i don't want to do this because this way my music it's maniacally ordered in lollypop on my pc which is exactly how music should be organized to my tastes...

Lollypop works exactly the contrary of what you told me, and to me this feel more right, simply because album artist  should be the artist who done the album, so no feat required while artist should be the artist who done the song...in this case the feat is required

Il Domenica 31 dicembre 2017, d9h20f scrive:
> You can solve this yourself by setting the album artist tag to 'Goo Goo Dolls', and fixing the artist tag of 'Goo Goo Dolls feat. Sydney Sierota' to a double artist tag: 'Goo Goo Dolls' and 'Sydney Sierota'. You can easily do this with Ex Falso (https://github.com/quodlibet/quodlibet), not sure how to do it with other tag editors. You will be able to find all 11 tracks under Goo Goo Dolls, and 'Flood' under Sydney Sierota, too.
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub, or mute the thread.￼
>

-- 
Inviato dal mio Jolla

**Comment by ghost** [comment]
No, it works exactly the way I described, because I suggested that behaviour in the first place:

![example](https://i.imgur.com/sLGtsUp.png)

This album has Andrew Ashong as albumartist, Andrew Ashong as artist on all tracks, and both Andrew Ashong and Theo Parrish as artist on the first track. No 'feat.' necessary.

**Comment by carlosgonz0** [comment]
I think this bug or mess-musical could be fixed by using a tag-editor @legacychimera247.
